src/native/eventpipe/ep-session.h

```diff
@@ -71,6 +71,9 @@ struct _EventPipeSession_Internal {
 	volatile uint32_t ref_count;
 	// The user_events_data file descriptor to register Tracepoints and write user_events to.
 	int user_events_data_fd;
+	// The IPC continuation stream from initializing the session through the diagnostic server
+	// Currently only initialized for user_events sessions.
+	IpcStream *stream;
```

Comment:
We don't need to do it in this PR, but going forward I believe we should do different versions of this struct depending on the type and move specific fields into sub types instead of having a fields in one struct where type dictates what fields is applicable without any further information.

Reply:
Right, I think we can have a Refactoring PR where we split up the versions of the struct, also migrate the IPC types to its own file, and move the user_events_data fd into the Stream structure

Reply:
So [IPC namespace](https://github.com/dotnet/runtime/pull/117435/#discussion_r2199927716), [Clarify the IpcStream inheritance naming](https://github.com/dotnet/runtime/pull/117435/#discussion_r2202296190), this, and the followup work from the main user_events PR (https://github.com/dotnet/runtime/pull/115265#discussion_r2139467105 ,https://github.com/dotnet/runtime/pull/115265#discussion_r2123840111, https://github.com/dotnet/runtime/pull/115265#discussion_r2123349055)